explorer/client/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -0,0 +1,196 @@
+        frontChars = Math.ceil(charsToShow / 2),
+        backChars = Math.floor(charsToShow / 2);
+
+    return (
+        fullStr.substr(0, frontChars) +
+        separator +
+        fullStr.substr(fullStr.length - backChars)
+    );
+}
+
+function LastestTxCard() {
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [results, setResults] = useState(initState);
+    useEffect(() => {
+        let isMounted = true;
+        getRecentTransactions(15)
+            .then((resp: any) => {
+                if (isMounted) {
+                    setIsLoaded(true);
+                }
+                setResults({
+                    loadState: 'loaded',
+                    lastestTx: resp,
+                });
+            })
+            .catch((err) => {
+                setResults({
+                    ...initState,
+                });
+            });
+
+        return () => {
+            isMounted = false;
+        };
+    }, []);
+    if (!isLoaded) {
+        return (
+            <div className={theme.textresults}>
+                <div className={styles.content}>Loading...</div>
+            </div>
+        );
+    }
+
+    if (!isLoaded && results.loadState === 'fail') {
+        return (
+            <ErrorResult
+                id="lastestTx"
+                errorMsg="There was an issue getting the lastest transaction"
+            />
+        );
+    }
+
+    return (
+        <div className={styles.txlatestesults}>
+            <div className={styles.txcardgrid}>
+                <h3>Latest Transaction</h3>
+            </div>
+            <div className={styles.txsearch}>{isLoaded && <Search />}</div>
+            <div className={styles.transactioncard}>
+                <div>
+                    <div
+                        className={cl(
+                            styles.txcardgrid,
+                            styles.txcard,
+                            styles.txheader
+                        )}
+                    >
+                        <div className={styles.txcardgridlarge}>TxId</div>
+                        <div className={styles.txtype}>Tx Type</div>
+                        <div className={styles.txgas}>Gas</div>
+                        <div className={styles.txadd}>Sender & Reciever</div>
+                    </div>
+                    {results.lastestTx.map((tx: any, index: number) => (
+                        <div
+                            key={index}
+                            className={cl(styles.txcardgrid, styles.txcard)}
+                        >
+                            <div className={styles.txcardgridlarge}>
+                                <div className={styles.txlink}>
+                                    <Longtext
+                                        text={tx.txId}
+                                        category="transactions"
+                                        isLink={true}
+                                        alttext={truncate(tx.txId, 26, '...')}
```

Comment:
why is this 26 and others are 25?

Reply:
@666lcz does not matter, its just fits(truncate) the `txId` and the `address` into their spaces